### ISSUE TITLE
Crusade and Battle Size interaction

### ISSUE THREAD

**Issue by BSDataAnon** [BSData/wh40k-9e#12140]
**File:** Warhammer 40,000.gstz

**BattleScribe version:** 2.03.25

**Platform:** Android

**Dropbox:** No

**Description:** In the Crusade game type selecting battle size gives an error saying you can't have a battle size. Removing battle size gives an error saying you must have a  battle size

**Comment by Valorcalls** [comment]
Crusade game type still uses command points, so I'm not sure what the error here is?

**Comment by felixlinker** [comment]
I think I encountered the same bug when building an unbound army (example attached). When loading the roster attached, you get the error:
<img width="491" alt="image" src="https://user-images.githubusercontent.com/9728715/212567012-ef40b4a3-c42b-4742-a4b6-c82ba91fe1b8.png">

However, when I remove the battle size, I get the error:
<img width="497" alt="image" src="https://user-images.githubusercontent.com/9728715/212567026-0d4db8bd-a821-4464-89f2-8466012e5de7.png">

In other words, battle size minimum is greater than battle size maximum.

[exm.txt](https://github.com/BSData/wh40k/files/10421201/exm.txt)

**Comment by WindstormSCR** [comment]
Unable to reproduce the issue of using narrative (crusade) and battle size with several catalogs and getting errors, except if there are multiple detachments present.

**Comment by felixlinker** [comment]
@WindstormSCR Would you prefer me opening a new issue with my "sub-issue"?

**Comment by WindstormSCR** [comment]
> @WindstormSCR Would you prefer me opening a new issue with my "sub-issue"?

please, it'll remind me to fix that later without closing this one

**Comment by MechMK1** [comment]
@WindstormSCR I'm experiencing the same as @felixlinker, with multiple catalogues on unbound armies and v9.7.3. I've uploaded a minimal example .rosz (as .zip), which demonstrates where this issue occurs.

Is any more information needed to replicate this?


[Ork Battle Size Error.zip](https://github.com/BSData/wh40k/files/10505399/Ork.Battle.Size.Error.zip)
